dev-tools/ecs-migration.yml

```diff
@@ -891,6 +891,28 @@
   to: network.forwarded_ip
   alias: false
 
+## NFS
+- from: rpc.time
+  to: event.duration
+  alias: false
+  comment: >
+    Cannot add alias at this time because the field is used by MongoDB which
+    has not been updated yet.
```

Comment:
Also because the scale is different, right? Visually in Kibana it should be fine, but it may lead to surprises if people use this field otherwise, like for thresholds. E.g. `rpc.time > 1000`. After the move to nanoseconds, this would match pretty much everything ;-)

Reply:
The two below I agree can be turned into actual aliases later, however.

Reply:
Good point. The same applies to responsetime and event.duration since they are different units.